app/views/ui-components/v1/filters/_plan_filters.html.erb

```diff
@@ -2,175 +2,6 @@
 <% if @bs4 %>
   <div id="filter-sidebar" class="w-100">
     <% is_ivl_osse_filter_eligible = ivl_osse_filtering_enabled? && @hbx_enrollment.ivl_osse_eligible? %>
-
```

Comment:
is this code removal expected with this PR? or just the single key change?

Reply:
i did the cleanup since I was in there anyway, the only pertinent change is the switch of the key